setup.py

```diff
@@ -38,12 +38,35 @@ def read_version():
     "numpy>=1.9.0",
     "protobuf>=3.1",
     "scipy>=0.19.0",
-    "urllib3>=1.21, <1.25",
     "protobuf3-to-dict>=0.1.5",
     "requests>=2.20.0, <2.21",
     "fabric>=2.0",
 ]
 
+# Specific use case dependencies
+extras = {"analytics": ["pandas"], "local": ["urllib3>=1.21, <1.25", "docker-compose>=1.23.0"]}
```

Comment:
should we add one for `tensorflow`? (if people want to use the TF serializers/deserializers that require the library itself)

Reply:
i didn't want to touch it, but actually why not :)